packages/firestore/android/src/reactnative/java/io/invertase/firebase/firestore/ReactNativeFirebaseFirestoreCollectionModule.java

```diff
@@ -214,6 +312,8 @@ public void collectionGet(
             orders,
             options);
     handleQueryGet(firestoreQuery, getSource(getOptions), promise);
+
+
```

Comment:
low-value comment: that should fail `yarn lint:android` shouldn't it? (but just by running lint:android it'll auto-format it and you can just commit result)